Source/Core/DolphinQt2/RenderWidget.cpp

```diff
@@ -6,17 +6,21 @@
 #include <QDesktopWidget>
 #include <QGuiApplication>
 #include <QKeyEvent>
+#include <QMouseEvent>
 #include <QPalette>
 #include <QScreen>
 #include <QTimer>
 
 #include "Core/ConfigManager.h"
 #include "Core/Core.h"
+#include "VideoCommon/VertexShaderManager.h"
+#include "VideoCommon/VideoConfig.h"
+
 #include "DolphinQt2/Host.h"
 #include "DolphinQt2/RenderWidget.h"
 #include "DolphinQt2/Settings.h"
 
-RenderWidget::RenderWidget(QWidget* parent) : QWidget(parent)
+RenderWidget::RenderWidget(QWidget* parent) : QWidget(parent), m_last_mouse{}
```

Comment:
The initialization can be done in the header. std::array<float, 2> m_last_mouse{}